README.md

```diff
@@ -34,31 +34,14 @@
 ```bash
 # clone repository
 git clone git@github.com:metisova/sturdy-lamp.git
-```
-
-### Start the server
-
-```bash
-# cd into server
-cd server
-
-# install server's dependencies
-npm install
-
-# run server
-npm run start:server
-```
 
-### Start the client
-
-```bash
-# open another terminal and cd into client
-cd client
+# go to app
+cd app
 
-# install client's dependencies
+# install dependencies
 npm install
 
-# run client
+# run the app
 npm run dev
```

Comment:
the serverless functions are now part of the main app, so I renamed `client` to `app`, and we now need just 1 command to run it, yay